integration/test_alertrules.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+
+@pytest.mark.skip("Failing with 404.")
```

Comment:
I think this is failing because you are rule id when you should be using the observer ID or visa-versa (I don't remember but I always do this).